src/support/kenniscloud_schema.erl

```diff
@@ -154,6 +155,12 @@ manage_data({upgrade, 20}, Context) ->
     remove_unused_predicate(hasattachment, Context),
     ok;
 manage_data({upgrade, 21}, _Context) ->
+    ok;
+manage_data({upgrade, N}, Context) when N >= 22 ->
+    SudoContext = z_acl:sudo(Context),
+    m_rsc:delete(region_utrecht, SudoContext),
+    m_rsc:delete(region_noordoost_brabant, SudoContext),
+    m_rsc:delete(region_dordrecht, SudoContext),
     ok.
```

Comment:
For this to work you'd also need to bump the `mod_schema` value in `kenniscloud` to 22, but I think that it can be avoided and we can remove this manually from acceptance for one more time (on production these are already gone AFAIK).